src/Renderer/Camera.js

```diff
@@ -174,4 +174,30 @@ Camera.prototype.getFrustumLocalSpace = function (position, quaternion) {
 };
 
 
+Camera.prototype.getRTCMatrixFromCenter = function (center) {
```

Comment:
These are `nodeProcess`, like occlusion, horizon occlusion...
The format of a process is `foo(node, camera, params)` to be able to be added generically in a `updater` or `browseTree`. The idea is to have a generic `updater` to which have added some `nodeProcess`, but not having several updater.
This is not the best place, because it is not the camera job to calculate a `RTC` matrix for a node. This should be a `nodeProcess` as a static function